client/draft_README.md

```diff
@@ -366,19 +366,19 @@ Gratitude to the following people who have contributed to this project:
 </div>
 
 <h3>Sergio Silva</h3>
-<a href="mailto:[email]@[provider]" target="_blank">
+<a href="mailto:sergiobtos@hotmail.com" target="_blank">
     <img src="https://img.shields.io/badge/-[email_address]-c14438?style=flat&logo=Gmail&logoColor=white">
  </a>
 <div style='display: flex; justify-content: center; gap: .5rem'>
 
-<a href="https://www.linkedin.com/in/username" target="_blank">
+<a href="https://www.linkedin.com/in/sergio-btos" target="_blank">
```

Comment:
Hi @sergiobtos! The provided link produces an error. See the error below. Can you please re-check and update?

“This page doesn’t exist. Please check your URL or return to LinkedIn home.”

Reply:
That is my linkedin, but for now I put on hold because I was receiving a lot job offers everyday, and I do not like to say no to recruiters. Because of that I put my profile on hold for now. My intention is put it back after january, so it will be the same link as you see above. For that reason, I would like to keep this link in our project. Can you accept that ?